server/shadow/Win/win_wds.c

```diff
@@ -764,21 +764,24 @@ int win_shadow_wds_init(winShadowSubsystem* subsystem)
 
 	WLog_INFO(TAG, "ConnectionString: %s", file->ConnectionString2);
 
-	if (0)
-	{
-		FILE* fp;
-		size_t size;
+#if 0
+	FILE* fp;
+	size_t size;
 
-		fp = fopen("inv.xml", "w+b");
+	fp = fopen("inv.xml", "w+b");
 
-		if (fp)
+	if (fp)
+	{
+		size = strlen(file->ConnectionString2);
+		if (fwrite(file->ConnectionString2, 1, size, fp) != size || fwrite("\r\n", 1, 2, fp) != 2)
```

Comment:
@nfedera size J?
I haven't really checked that code since it was if(0) but in order to not have it compiled in I disabled it properly with #if 0.